<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputMimicPose</class>
 <widget class="QWidget" name="InputMimicPose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelName">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelAnim">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>anim:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelEyes">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>eyes:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="5">
    <widget class="QLineEdit" name="lineName"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelPose">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>pose:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelState">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>emotional state:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="labelWeight">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>weight:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="labelDuration">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>duration (for change):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="5">
    <widget class="QComboBox" name="comboEyes">
     <property name="maxVisibleItems">
      <number>25</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="5">
    <widget class="QComboBox" name="comboPose">
     <property name="maxVisibleItems">
      <number>25</number>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="5">
    <widget class="QComboBox" name="comboState">
     <property name="maxVisibleItems">
      <number>25</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="5">
    <widget class="QComboBox" name="comboAnim">
     <property name="maxVisibleItems">
      <number>25</number>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="5">
    <widget class="QDoubleSpinBox" name="spinDuration">
     <property name="suffix">
      <string> s</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="5">
    <widget class="QDoubleSpinBox" name="spinWeight">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.001000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
